ovirtlib4/ovirtlib.py

```diff
@@ -31,6 +41,8 @@ def __init__(self, host, password, username=defaults.ADMIN_USERNAME, ca_file=Non
         self.password = password
         self.ca_file = ca_file
         self.insecure = insecure
+        self.logger = logger
```

Comment:
if the logger is with the default value(None) when we will try to use it we will get an error...
maybe in case of None, we need to initialize the log here(or in another method)

Reply:
None is also a default parameter for the 'log' parameter in the ovirtsdk itself so there's no danger in using it